examples/with-ecs-scheduling/main.tf

```diff
@@ -38,7 +41,12 @@ module "eventbridge" {
   create_role       = true
   role_name         = "ecs-eventbridge-${random_pet.this.id}"
   attach_ecs_policy = true
-  ecs_target_arns   = [module.ecs_cluster.services["hello-world"].task_definition_arn]
+  ecs_target_arns = [
+    module.ecs_cluster.services["hello-world"].task_definition_arn,
+    "arn:aws:ecs:${data.aws_region.current.id}:${data.aws_caller_identity.current.account_id}:task/${random_pet.this.id}/*"
+  ]
+
+  pass_role_resources = [module.ecs_cluster.services["hello-world"].task_exec_iam_role_arn]
```

Comment:
Let's rename this variable `pass_role_resources` to be scoped to ECS, for example, `ecs_pass_role_resources`